jest_config/test-utils.js

```diff
@@ -0,0 +1,34 @@
+// Setup react-testing-library
+// https://testing-library.com/docs/react-testing-library/setup#custom-render
+import { render } from '@testing-library/react'
+
+// Mock for features used by react-slider
+window.matchMedia =
+  window.matchMedia ||
+  function () {
+    return {
+      matches: false,
+      addListener: function () { },
+      removeListener: function () { }
+    };
+  };
+
+window.requestAnimationFrame =
+  window.requestAnimationFrame ||
+  function (callback) {
+    setTimeout(callback, 0);
+  };
+
+// Create custom renderer to share setup between component tests.
+const AllTheProviders = ({ children }) => {
+  return (<></>)
+}
+
+const customRender = (ui, options) => render(ui, { wrapper: AllTheProviders, ...options })
+
+
+// re-export everything
+export * from '@testing-library/react'
+
+// override render method
+// export { customRender as render }
```

Comment:
Is this planned for another PR? Currently `customRender` is not exported.

Reply:
Yes it's in anticipation for when we start testing components and need to wrap all the providers.